CITATION.cff

```diff
@@ -0,0 +1,15 @@
+cff-version: 1.2.0
+message: "If you use SWMManywhere, please cite it as below."
+authors:
+  - family-names: "Dobson"
+    given-names: "Barnaby"
+    orcid: "https://orcid.org/0000-0002-0149-4124"
+  - family-names: "Alonso Alvarez"
+    given-names: "Diego"
+  - family-names: "Chegini"
+    given-names: "Taher"
```

Comment:
@cheginit should also appear as author here: https://github.com/ImperialCollegeLondon/SWMManywhere/blob/f00cb38805f9a90fd41e55abf9f70f64cc9ca21f/pyproject.toml#L12

And for consistency, while we normally like to be included as the `RSE Team`, maybe I should also appear there with my name and email rather than the general RSE Team.

Reply:
should I remove RSE and add you? or do both?

Reply:
Just leave me.